content/master/cli/_index.md

```diff
@@ -61,4 +61,21 @@ By default the CLI installs from the `XP_CHANNEL` named `stable` and the
 For example, to install CLI version `v1.14.0` add `XP_VERSION=v1.14.0` to the 
 download script curl command:  
 
-`curl -sL "https://raw.githubusercontent.com/crossplane/crossplane/main/install.sh" | XP_VERSION=v1.14.0 sh`
\ No newline at end of file
+`curl -sL "https://raw.githubusercontent.com/crossplane/crossplane/main/install.sh" | XP_VERSION=v1.14.0 sh`
+
+## Installing shell autocompletions
+
+The Crossplane CLI supports shell autocompletions for `bash`, `zsh` and `fish`.
+You can install the autocompletions with the `completions` command by adding it to
+your shell's configuration file.
+
+```shell
+source <(crossplane completions)
+```
+
+{{<hint "note" >}}
+The `completions` command generates the autocompletions for your default shell.
+It's not possible to generate autocompletions for a different shell, if you want to
+install the autocompletions for a different shell, you have to configure the crossplane
```

Comment:
I think this crossplane mention needs to be capitalized, since it's not in a code block. I think that's what's causing the vale error in https://github.com/crossplane/docs/actions/runs/14866616811/job/41744888843?pr=928, even though the line number seems off. I made this change locally and it made the vale error go away 😂 

More about working with Vale: https://docs.crossplane.io/contribute/vale/

Note, only errors must be fixed, warnings are OK if we don't think it's worth fixing them.

Reply:
Actually it's because of the capitalization. If `Crossplane` is capitalized vale seems to treat it is a name and doesn't complain